x/group/server/invariants.go

```diff
@@ -0,0 +1,96 @@
+package server
+
+import (
+	"fmt"
+	"math"
+
+	"github.com/regen-network/regen-ledger/orm"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/regen-network/regen-ledger/types"
+	"github.com/regen-network/regen-ledger/x/group"
+)
+
+func (s serverImpl) RegisterInvariants(ir sdk.InvariantRegistry) {
```

Comment:
```suggestion
func (s serverImpl) RegisterInvariantsHandler(ir sdk.InvariantRegistry) {
```

Reply:
We should not change RegisterInvariants since this is a part of AppModule interface